### ISSUE TITLE
A11y - Page layout lacks a11y landmarks

### ISSUE THREAD

**Issue by luistoptal** [gigascience/gigadb-website#1361]
## User story

As a user, I should identify page landmarks such as header, main, footer...

## Acceptance criteria

Page should have header, main and footer landmarks, appropriately used

## Additional Info

Page layout used in all pages should probably be the file adequately incorporating semantinc landmarks

Reference:

https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/
https://www.scottohara.me/blog/2018/03/03/landmarks.html
